tests/resources/Vsphere-Util.robot

```diff
@@ -277,6 +277,16 @@ Get Hostd Memory Consumption
     Close Connection
     [Return]  ${out}
 
+# NOTE: this function should only be used in the case where you supply a name to the container. Since inventory names result in `<cvm-name>-<cvm-uuid>`
+Check CVM Inventory Path
```

Comment:
I also want to note for reviewers. This function will be very useful and heavily used for the vic-machine inventory integration tests. So it is also some future prep.